AutomatedTesting/Gem/PythonTests/EditorPythonTestTools/editor_python_test_tools/QtPyScriptCanvasNodePalette.py

```diff
@@ -9,8 +9,7 @@
 from editor_python_test_tools.utils import TestHelper as helper
 from PySide2 import QtWidgets, QtTest, QtCore
 import pyside_utils
-import azlmbr.legacy.general as general
-from consts.scripting import (NODE_PALETTE_QT, TREE_VIEW_QT, SEARCH_FRAME_QT, SEARCH_FILTER_QT)
+from consts.scripting import (NODE_PALETTE_QT, TREE_VIEW_QT, SEARCH_FRAME_QT, SEARCH_FILTER_QT, NODE_PALETTE_CLEAR_BUTTON_QT)
 from consts.general import (WAIT_TIME_SEC_1, WAIT_TIME_SEC_3)
 
 class QtPyScriptCanvasNodePalette():
```

Comment:
Not necessarily in scope of this PR, but this node palette is actually in the base `GraphCanvas` gem and is used by our other *Canvas tools as well (e.g. Landscape Canvas, Material Canvas, etc...) so might be worth renaming this to `QtPyGraphCanvasNodePalette`